Tasks/PublishSymbols/PublishSymbols.ps1

```diff
@@ -11,6 +11,9 @@ param()
 # .\_build\Tasks\PublishSymbols\PublishSymbols.ps1
 
 Trace-VstsEnteringInvocation $MyInvocation
+
+$ErrorActionPreference = "Stop"
```

Comment:
@arunkm Would either of these change existing behavior on-prem?

Reply:
Technically Yes, if there was an error that was always resumed before, this will now terminate and fail the script.